src/client.ts

```diff
@@ -8794,368 +8764,6 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         return new MSC3089TreeSpace(this, roomId);
     }
 
-    // TODO: Remove this warning, alongside the functions
-    // See https://github.com/vector-im/element-web/issues/17532
-    // ======================================================
-    // **                ANCIENT APIS BELOW                **
-    // ======================================================
```

Comment:
I would like to point out that code got added below this warning that is not subject to the "ancient apis" warning. Not sure if we should do a better job communicating these things in the future.